todo.js

```diff
@@ -0,0 +1,14 @@
+let inputs = document.getElementById("inp")
```

Comment:
1. We should use `const` instead of `let` if we don't reassign the variable.
2. Variable naming like function naming should be meaningful. For variables it should clearly denote what it contains or what it points to. In this case let's use `inputElement`. Notice that I've prefixed the name with the word `element` which makes it clear that the variable contains reference to a dom input element.
3. 
```suggestion
const inputElement = document.getElementById("todoInput")
```